### PR TITLE
Add save command

### LINKED ISSUE #153: As a user I can save file into a folder of my choice

### PR DESCRIPTION
**Pull request by l0g1cal**
Fixes #153

### CODE REVIEW

File: src/main/java/seedu/taskitty/logic/commands/SaveCommand.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * @author JiaWern
+ *
+ */
+public class SaveCommand extends Command{
+    
+    public static final String COMMAND_WORD = "save";
+
+    public static final String MESSAGE_PARAMETER = COMMAND_WORD
+            + " filepath";
+    public static final String MESSAGE_USAGE = "This command saves data in TasKitty to a location of your choice, Meow!\n"
```
Comment (IanTeo) [inline src/main/java/seedu/taskitty/logic/commands/SaveCommand.java]: MESSAGE_USAGE is too long for us to fit. Try to cut it down to 1 or 2 lines. Just basic info. The rest of the info, the user will have to check the UserGuide

File: src/main/java/seedu/taskitty/logic/commands/SaveCommand.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    
+    public SaveCommand(String filepath) throws IllegalValueException{
+        if (filepath.toCharArray().length == 0) {
+            throw new IllegalValueException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, MESSAGE_USAGE));
+        };
+        this.filepath = filepath;
+    }
+
+    @Override
+    public CommandResult execute() {
```
Comment (IanTeo) [inline src/main/java/seedu/taskitty/logic/commands/SaveCommand.java:40]: Just wondering.. But does this save TasKitty into the folder specified once?
Or does it change the save folder destination permanently?

**Review by IanTeo — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/taskitty/logic/commands/SaveCommand.java (hunk 2 above)
Comment (l0g1cal) [reply src/main/java/seedu/taskitty/logic/commands/SaveCommand.java:40]: It changes it permanently, based on my tests

Comment (IanTeo) [reply src/main/java/seedu/taskitty/logic/commands/SaveCommand.java:40]: Ok great

**Review by IanTeo — APPROVED** [review]

**Comment by IanTeo** [comment]
Jia Wern, you can fix the merge conflicts and merge it.
I feel that "save" might be ambiguous for this command.
The user might feel that it is to save TasKitty, not choose the save location.
I can't really think of a better command though.. "storage"? Anyone else have comments? @yamidark  @JJ-Rachel

**Comment by l0g1cal** [comment]
How about 'store'?

**Comment by IanTeo** [comment]
fixes #155

**Pull request merged**